packages/core/core/src/factories/useWishlistFactory.ts

```diff
@@ -66,9 +66,10 @@ export const useWishlistFactory = <WISHLIST, WISHLIST_ITEM, PRODUCT>(
     const loadWishlist = async (customQuery?: CustomQuery) => {
       Logger.debug('useWishlist.loadWishlist');
 
+      loading.value = true;
```

Comment:
Maybe it's insignificant, but what for invoke loading if you already have a wishlist? 

```suggestion
      if (wishlist.value) return;
      loading.value = true;
```

Reply:
we had discussions about this with @andrzejewsky and as far as I remember we decided that it's better to leave it like this to avoid confusion with debugging

you can do similar in your code:

```js
onSSR(async () {
 if (wishlist.value) await loadWishlist()
})
```

Reply:
Removed it